set.mm

```diff
@@ -609582,6 +609582,66 @@ fixed reference functional determined by this vector (corresponding to
 #*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
 $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Various arithmetic theorems
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $( Alternate version of ~ 1t1e1 using a different set of axioms.
+     (Contributed by Steven Nguyen, 20-Sep-2022.)
+     (Proof modification is discouraged.)  (New usage is discouraged.) $)
+  1t1e1ALT $p |- ( 1 x. 1 ) = 1 $=
+    ( c1 cr wcel cmul co wceq 1re ax-1rid ax-mp ) ABCAADEAFGAHI $.
+
+  ${
+    sqsumi.1 $e |- A e. CC $.
+    sqsumi.2 $e |- B e. CC $.
+    $( A sum squared.  (Contributed by Steven Nguyen, 16-Sep-2022.) $)
+    sqsumi $p |- ( ( A + B ) x. ( A + B ) ) =
+      ( ( ( A x. A ) + ( B x. B ) ) + ( 2 x. ( A x. B ) ) ) $=
+      ( caddc co cmul c2 muladdi mulcli 2timesi eqcomi oveq2i eqtri ) ABEFZOGFA
+      AGFBBGFEFZABGFZQEFZEFPHQGFZEFABABCDCDIRSPESRQABCDJKLMN $.
+  $}
+
+  ${
+    diffsqi.a $e |- A e. CC $.
+    diffsqi.n $e |- N e. CC $.
+    diffsqi.b $e |- ( A + N ) = B $.
+    diffsqi.c $e |- ( B + N ) = C $.
+    $( Difference of squares.  (Contributed by Steven Nguyen, 20-Sep-2022.) $)
+    diffsqi $p |- ( B x. B ) = ( ( A x. C ) + ( N x. N ) ) $=
```

Comment:
Where is the "difference"? ( ( B x. B ) - ( N x. N ) ) = ( A x. C )?

B^2 = A^2 + 2AN + N^2 
=> B^2 - N^2 = A^2 + 2AN = A ( A + 2N). Why not choosing C = ( A + ( 2 x. N ) ) directly? See also comment of @benjub about ~sqn5ii

Reply:
Really I created the theorem first and then the name/comments. I'm not sure how to describe this theorem

Reply:
"Value of the square of the midterm of a 3-term arithmetic progression." ?